phenol-io/src/test/java/org/monarchinitiative/phenol/io/PrepareToyHpoJsonTest.java

```diff
@@ -0,0 +1,64 @@
+package org.monarchinitiative.phenol.io;
+
+import org.monarchinitiative.phenol.ontology.data.Ontology;
+import org.monarchinitiative.phenol.ontology.data.TermId;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * The class serves for extracting terms for a small HP ontology JSON <em>hpo_toy.json</em> containing ancestors of
+ * <a href="https://hpo.jax.org/app/browse/term/HP:0001166">Arachnodactyly</a> for the
+ * <a href="https://hpo.jax.org/app/browse/term/HP:0000118">Phenotypic abnormality</a> sub-hierarchy, and all the other
+ * sub-hierarchies of <em>All</em>, including:
+ * <ul>
+ *   <li>Clinical modifier</li>
+ *   <li>Mode of inheritance</li>
+ *   <li>Past medical history</li>
+ *   <li>Blood group</li>
+ *   <li>Frequency</li>
+ * </ul>
+ *
+ * <p>
+ * As Surefire warns regarding disabled tests during build, the JUnit test annotation is commented out to disable
+ * the warning. Run the test manually if necessary.
+ */
+public class PrepareToyHpoJsonTest {
+
+//  @Test
```

Comment:
Does this need re-instating, along with adding the missing path?

Reply:
Hard to say now. I think presence of the file does not cost a lot. It's a kind of a "Java"-script.

Reply:
It's more a case of  - is this required for something or has that something now outlived its purpose and needs to bee deleted.